### ISSUE TITLE
All depends my *

### ISSUE THREAD

**Issue by xXXIMMATTHEWXXx** [jcabi/jcabi-github#1269]
The "jcabi-github" with all depends does _not_ contain all needed dependence. The consoles throws some error about missing a static logger class.

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by yegor256** [comment]
@xXXIMMATTHEWXXx what exactly is the exception you're getting?

**Comment by xXXIMMATTHEWXXx** [comment]
@yegor256 this:
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```
is thrown

**Comment by yegor256** [comment]
@xXXIMMATTHEWXXx it's just a warning in logging console, not an exception. You can just ignore this.

**Comment by GIAPB** [comment]
Any way to make it go away. I dont think it will be nice for storage over a very long time as it happens every time I launch it

**Comment by yegor256** [comment]
@GIAPB to make it disappear you have to configure SLF4J bridge to some logging facility, like log4j.

**Comment by xXXIMMATTHEWXXx** [comment]
This _should_ be the in docs

**Comment by yegor256** [comment]
@xXXIMMATTHEWXXx I agree. Would be great if you can submit a pull request. I would suggest to make changes to README.md. This will be helpful: http://stackoverflow.com/questions/4311026/how-to-get-slf4j-hello-world-working-with-log4j
